voltage/__init__.py

```diff
@@ -16,7 +16,7 @@
 __author__ = "EnokiUN"
 __license__ = "MIT"
 __copyright__ = "Copyright (c) 2021-present EnokiUN"
-__version__ = "0.1.5a8"  # Updating this is such a pain.
+__version__ = "0.1.6a"  # Updating this is such a pain.
```

Comment:
You also probably want to update that here <https://github.com/EnokiUN/voltage/blob/bcd308f94a3eff1f9227a66f98e212ce6886783a/voltage/internals/ws.py#L105> (because it's hardcoded for some reason (why did I do this) :trollface:)